js_modules/dagit/packages/core/src/app/getVisibleFeatureFlagRows.tsx

```diff
@@ -13,11 +13,11 @@ export const getVisibleFeatureFlagRows = () => [
     flagType: FeatureFlag.flagDisableWebsockets,
   },
   {
-    key: 'Experimental schedule/sensor logging view',
-    flagType: FeatureFlag.flagSensorScheduleLogging,
+    key: 'Display resources in sidebar',
```

Comment:
Will it be clear to the user what the "sidebar" is? Is that the assets sidebar?

Reply:
The navigation sidebar on the left. Will update to be a bit more descriptive!